src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java

```diff
@@ -159,6 +160,42 @@ default FlowResponsePayload toFlowResponseOutput(FlowReadResponse r) {
     @Mapping(target = "type", ignore = true)
     FlowRequest toFlowRequest(FlowRequestV2 request);
 
+
+    @Mapping(target = "flowId", source = "id")
+    @Mapping(target = "sourceSwitch", expression = "java(request.getSource().getSwitchDpId())")
+    @Mapping(target = "destinationSwitch", expression = "java(request.getDestination().getSwitchDpId())")
+    @Mapping(target = "sourcePort", expression = "java(request.getSource().getPortNumber())")
+    @Mapping(target = "destinationPort", expression = "java(request.getDestination().getPortNumber())")
+    @Mapping(target = "sourceVlan", expression = "java(request.getSource().getVlanId())")
+    @Mapping(target = "destinationVlan", expression = "java(request.getDestination().getVlanId())")
+    @Mapping(target = "bandwidth", source = "maximumBandwidth")
+    @Mapping(target = "transitEncapsulationId", ignore = true)
+    @Mapping(target = "type", ignore = true)
+    @Mapping(target = "detectConnectedDevices", ignore = true)
+    FlowRequest toFlowRequest(FlowCreatePayload request);
+
+    /**
+     * Map FlowCreatePayload.
+     *
+     * @param source {@link FlowCreatePayload} instance.
+     * @return {@link FlowRequest} instance.
+     */
+    default FlowRequest toFlowCreateRequest(FlowCreatePayload source) {
+        FlowRequest target = toFlowRequest(source).toBuilder().type(Type.CREATE).build();
+        DetectConnectedDevicesDto detectConnectedDevicesDto = new DetectConnectedDevicesDto();
+        if (source.getSource().getDetectConnectedDevices() != null) {
```

Comment:
I would like to have a copy constructor for `DetectConnectedDevicesDto` or mapper method to copy such objects (at least it remove code duplication).

Reply:
in v1 detect info was part of flow endpoint, in v2 it's a separate entity. 
v1: FlowCreatePayload::FlowEndpointPayload::DetectConnectedDevicesPayload(separation based on endpoint level
v2: FlowRequest::DetectConnectedDevicesDto - single object, separation based on field level, 
e.g. srcLldp/dstLldp

Does it make sense to implement such mapper? Could see little gain here.